contrib/cirrus/runner.sh

```diff
@@ -67,6 +67,8 @@ _run_unit_aarch64() {
 }
 
 _run_integration() {
+	dnf -y install dnsmasq jq
```

Comment:
Is there a plan to put this into the VM images, jq should be already installed?

Reply:
yup, right after this pr merges